components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java

```diff
@@ -2563,44 +2536,12 @@ public static class GlobalCacheInvalidation {
 
     // Supported Notifier Types
     public enum NotifierType {
-        API,
-        GATEWAY_PUBLISHED_API,
-        APPLICATION,
-        APPLICATION_REGISTRATION,
-        POLICY,
-        SUBSCRIPTIONS,
-        SCOPE,
-        CERTIFICATE,
-        GA_CONFIG
+        API, GATEWAY_PUBLISHED_API, APPLICATION, APPLICATION_REGISTRATION, POLICY, SUBSCRIPTIONS, SCOPE, CERTIFICATE, APPLICATION_CERTIFICATE, GA_CONFIG
     }
 
     // Supported Event Types
     public enum EventType {
-        API_CREATE,
-        API_UPDATE,
-        API_DELETE,
-        API_LIFECYCLE_CHANGE,
-        APPLICATION_CREATE,
-        APPLICATION_UPDATE,
-        APPLICATION_DELETE,
-        APPLICATION_REGISTRATION_CREATE,
-        POLICY_CREATE,
-        POLICY_UPDATE,
-        POLICY_DELETE,
-        SUBSCRIPTIONS_CREATE,
-        SUBSCRIPTIONS_UPDATE,
-        SUBSCRIPTIONS_DELETE,
-        DEPLOY_API_IN_GATEWAY,
-        REMOVE_API_FROM_GATEWAY,
-        REMOVE_APPLICATION_KEYMAPPING,
-        SCOPE_CREATE,
-        SCOPE_UPDATE,
-        SCOPE_DELETE,
-        ENDPOINT_CERTIFICATE_ADD,
-        ENDPOINT_CERTIFICATE_UPDATE,
-        ENDPOINT_CERTIFICATE_REMOVE,
-        GA_CONFIG_UPDATE,
-        UDATE_API_LOG_LEVEL
+        API_CREATE, API_UPDATE, API_DELETE, API_LIFECYCLE_CHANGE, APPLICATION_CREATE, APPLICATION_UPDATE, APPLICATION_DELETE, APPLICATION_REGISTRATION_CREATE, POLICY_CREATE, POLICY_UPDATE, POLICY_DELETE, SUBSCRIPTIONS_CREATE, SUBSCRIPTIONS_UPDATE, SUBSCRIPTIONS_DELETE, DEPLOY_API_IN_GATEWAY, REMOVE_API_FROM_GATEWAY, REMOVE_APPLICATION_KEYMAPPING, SCOPE_CREATE, SCOPE_UPDATE, SCOPE_DELETE, ENDPOINT_CERTIFICATE_ADD, ENDPOINT_CERTIFICATE_UPDATE, ENDPOINT_CERTIFICATE_REMOVE, APPLICATION_CERTIFICATE_ADD, APPLICATION_CERTIFICATE_REMOVE, GA_CONFIG_UPDATE, UDATE_API_LOG_LEVEL
```

Comment:
this line is too long. any reason for formatting this whole class? can you only add your changes plz